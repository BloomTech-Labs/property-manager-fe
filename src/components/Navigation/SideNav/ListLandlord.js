import React from 'react';
import ListItem from '@material-ui/core/ListItem';
import ListItemIcon from '@material-ui/core/ListItemIcon';
import ListItemText from '@material-ui/core/ListItemText';
import HomeWorkIcon from '@material-ui/icons/HomeWork';
import AccountCircleIcon from '@material-ui/icons/AccountCircle';
import PeopleIcon from '@material-ui/icons/People';
import { navigate } from '@reach/router';

function ListView() {
  return (
    <div>
      <ListItem button onClick={() => navigate('/dashboard/profile')}>
        <ListItemIcon>
          <AccountCircleIcon />
        </ListItemIcon>
        <ListItemText primary="Profile" />
      </ListItem>

      <ListItem button onClick={() => navigate('/dashboard/properties')}>
        <ListItemIcon>
          <HomeWorkIcon />
        </ListItemIcon>
        <ListItemText primary="Properties" />
      </ListItem>

      <ListItem button onClick={() => navigate('/dashboard/tenants')}>
        <ListItemIcon>
          <PeopleIcon />
        </ListItemIcon>
        <ListItemText primary="Tenants" />
      </ListItem>
    </div>
  );
}

export default ListView;
