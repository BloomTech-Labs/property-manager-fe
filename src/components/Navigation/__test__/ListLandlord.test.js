import React from 'react';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import ListLandlord from '../SideNav/ListLandlord';

afterEach(rtl.cleanup);

test('snapshot ListLandlord', () => {
  const { wrapper } = rtl.render(<ListLandlord />);
  expect(wrapper).toMatchSnapshot();
});

test('Renders', () => {
  const { wrapper } = rtl.render(<ListLandlord />);
  expect(wrapper).not.toBeNull();
});
