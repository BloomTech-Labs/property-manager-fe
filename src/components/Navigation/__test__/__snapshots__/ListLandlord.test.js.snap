// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot ListLandlord 1`] = `undefined`;
